azure-quantum/tests/unit/test_job.py

```diff
@@ -127,13 +127,15 @@ def test_job_submit_microsoft_population_annealing(self):
         solver_type = functools.partial(microsoft.PopulationAnnealing, sweeps=200)
         solver_name = "PopulationAnnealing"
         self._test_job_submit(solver_name, solver_type)
-        self._test_job_submit(solver_name, solver_type, test_grouped=True)
+        # todo - re-enable once grouped model is live
+        #self._test_job_submit(solver_name, solver_type, test_grouped=True)
 
     def test_job_submit_microsoft_substochastic_monte_carlo(self):
         solver_type = functools.partial(microsoft.SubstochasticMonteCarlo, step_limit=280)
         solver_name = "SubstochasticMonteCarlo"
         self._test_job_submit(solver_name, solver_type)
-        self._test_job_submit(solver_name, solver_type, test_grouped=True)
+        # todo - re-enable once grouped model is live
+        #self._test_job_submit(solver_name, solver_type, test_grouped=True)
```

Comment:
```suggestion
        self._test_job_submit(solver_name, solver_type, test_grouped=False)
```

Reply:
I didn't want to do this because it just runs the exact same test as the line above (so slightly redundant)